modules/hosted-control-planes-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-overview_{context}"]
+= Hosted control planes overview (technology preview)
+
+Hosted control planes for {product-title} provides a way to decouple your control plane from your worker nodes so that you can work in a hybrid cloud approach. You can enable hosted control planes through the multicluster engine for Kubernetes operator version 2.0. In that version, only Amazon Web Services (AWS) hosted clusters are available.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+== Hosted control planes architecture
+
+Typically in {product-title}, a cluster consists of a control plane and a data plane. The control plane includes an API endpoint, a storage endpoint, a workload scheduler, and an actuator that ensures state. The data plane is the hot path, including compute, storage, and networking where workloads and applications run.
+
+The control plane and the data plane are coupled in terms of locality. The control plane is hosted by a dedicated group of nodes, which can be physical or virtual, with a minimum number to ensure quorum. The network stack is shared, and admin access to a cluster offers visibility into the cluster's control plane, machine management APIs, and other system components that contribute to the state of a cluster.
+
+While the coupled model works well, some situations require an architectural model where the control plane and data plane ar decoupled. In the decoupled model, the data plane is on a separate network domain with a dedicated physical hosting environment. The control plane is hosted by using high-level primitives such as deployments and stateful sets that are native to Kubernetes.
+
+image::hosted-control-planes-diagram.png[Diagram that compares the hosted control plane model against OpenShift with a coupled control plane and workers]
+
+== Benefits of hosted control planes
+
+Hosted control planes for {product-title} paves the way for a hybrid-cloud approach and offers several other benefits.
+
+The management plane for control planes and cloud credentials is separate from the cluster that you interact with. Because the control plane is managed, you are less likely to accidentally delete your clusters.
```

Comment:
Do we want to bullet the benefits?

Reply:
By decoupling the control-plane and hosting them away on a dedicated hosting service cluster, hosted control planes hardens the security boundaries between management and workloads. For example, it is less likely to leak credentials to other users for the clusters. 

Furthermore, the decoupling infrastructure secret account management, as a result, as cluster infrastructure admin it is not possible to accidentally delete control-plane infrastructure.